src/js/components/about/AboutData.jsx

```diff
@@ -15,15 +15,8 @@ export default class AboutData extends React.Component {
                 <div className="aboutdata-inner-wrap">
```

Comment:
Can you provide verification that you want to launch this page with lorem ipsum on it?